python/dgl/distributed/dist_context.py

```diff
@@ -20,33 +20,37 @@ def set_initialized(value=True):
     global INITIALIZED
     INITIALIZED = value
 
-
 def get_sampler_pool():
     """Return the sampler pool and num_workers"""
     return SAMPLER_POOL, NUM_SAMPLER_WORKERS
 
-
-def _init_rpc(ip_config, max_queue_size, net_type, role, num_threads):
+def _init_rpc(ip_config, server_count, max_queue_size, net_type, role, num_threads):
     ''' This init function is called in the worker processes.
     '''
     try:
         utils.set_num_threads(num_threads)
-        connect_to_server(ip_config, max_queue_size, net_type)
+        connect_to_server(ip_config, server_count, max_queue_size, net_type)
         init_role(role)
-        init_kvstore(ip_config, role)
+        init_kvstore(ip_config, server_count, role)
     except Exception as e:
         print(e, flush=True)
         traceback.print_exc()
         raise e
 
-
-def initialize(ip_config, num_workers=0, max_queue_size=MAX_QUEUE_SIZE, net_type='socket',
+def initialize(ip_config, server_count=1, num_workers=0,
```

Comment:
i think it's better to rename `server_count` to `num_servers` to make it consistent with other argument names.